pkg/webui/locales/xx.json

```diff
@@ -63,34 +91,7 @@
-  "console.views.device-add.messages.createDevice": "Xxxxxx Xxxxxx",
-  "console.views.device-add.messages.deviceIdPlaceholder": "xx-xxx-xxxxxx",
-  "console.views.device-add.messages.deviceNamePlaceholder": "Xx Xxx Xxxxxx",
-  "console.views.device-add.messages.deviceIdDescription": "Xxxxxx Xxxxxxxxxx xx xxxx xxxxxx; xxxx xxxxxx xx xxxxxxx xxxxxxxxxx",
-  "console.views.device-add.messages.deviceNameDescription": "Xxxxx xxxxxxxx xxxx xx xxxx xxxxxx xxx xxxxxxx xxxxxxxx",
-  "console.views.device-add.messages.deviceDescDescription": "Xxxxxxxx xxxxxx xxxxxxxxxxx; xxx xxxx xx xxxx xx xxxx xxxxx xxxxx xxx xxxxxx",
-  "console.views.device-add.messages.joinEUIPlaceholder": "Xxx xxxxxxxxx Xxxx XXX",
-  "console.views.device-add.messages.leaveBlankPlaceholder": "Xxxxx xxxxx xx xxxxxxxx xxxxxxxxxxxxx",
-  "console.views.device-add.messages.resetsJoinNonces": "Xxxxxx Xxxx Xxxxxx",
-  "console.views.device-add.messages.resetsFCnt": "Xxxxxx Xxxxx Xxxxxxxx",
-  "console.views.device-add.messages.deviceEUIDescription": "Xxx xxxxxx XXX xx xxx xxxxxx xxxxxxxxxx xxx xxxx xxxxxx xx xxx xxxxxxx. Xxx xx xxxxxxx xxxxx.",
-  "console.views.device-add.messages.nwkKeyDescription": "Xxx xxxxxxxxx Xxxxxxx Xxx",
-  "console.views.device-add.messages.appKeyDescription": "Xxx Xxx Xxx xx xxxx xx xxxxxx xxx xxxxxxxxxxxxx xxxxxxx xxxx xxxxxx xxx xxx xxxxxxx.",
-  "console.views.device-add.messages.appSKeyDescription": "Xxx Xxxxxxx Xxx",
-  "console.views.device-add.messages.fwdNtwkKeyDescription": "Xxxxxxxxxx Xxxxxxx Xxxxxxx Xxxxxxxxx Xxx (xx XxXxXXX x.x.x XxxXXxx)",
-  "console.views.device-add.messages.sNtwkSIKeyDescription": "Xxxxxxx Xxxxxxx Xxxxxxx Xxxxxxxxx Xxx (xxxx xxx XxXxXXX x.x+)",
-  "console.views.device-add.messages.ntwkSEncKeyDescription": "Xxxxxxx Xxxxxxx Xxxxxxxxxx Xxx (xxxx xxx XxXxXXX x.x+)",
-  "console.views.device-add.messages.validate8": "Xxxx xxxxx xxxxx xx xx xxxxxxx x xxxxxxxxxx xxxx",
-  "console.views.device-add.messages.validate16": "Xxxx xxxxx xxxxx xx xx xxxxxxx xx xxxxxxxxxx xxxx",
-  "console.views.device-add.messages.validate32": "Xxxx xxxxx xxxxx xx xx xxxxxxx xx xxxxxxxxxx xxxx",
-  "console.views.device-add.messages.supportsClassC": "Xxxxxxxx Xxxxx X",
-  "console.views.device-add.messages.activationMode": "Xxxxxxxxxx Xxxx",
-  "console.views.device-add.messages.otaa": "Xxxx Xxx Xxx Xxxxxxxxxx (XXXX)",
-  "console.views.device-add.messages.abp": "Xxxxxxxxxx Xx Xxxxxxxxxxxxxxx (XXX)",
-  "console.views.device-add.messages.resetWarning": "Xxxxxxxx xx xxxxxxxx xxx xxxxx xxxx xxxxxx xxxxxxxxxxx xxx xxxxxx xxxxxxx",
-  "console.views.device-add.messages.couldNotRetrieveFrequencyPlans": "Xxxxx xxx xxxxxxxx xxx xxxx xx xxxxxxxxx xxxxxxxxx xxxxx",
+  "console.views.device-general-settings.index.updateSuccess": "Xxxxxxxxxxxx xxxxxxx xxx xxxxxx",
   "console.views.device-overview.index.hardware": "Xxxxxxxx",
   "console.views.device-overview.index.brand": "Xxxxx",
   "console.views.device-overview.index.model": "Xxxxx",
```

Comment:
please move these to `shared-messages.js`. The gateway model has them as well, see https://github.com/TheThingsNetwork/lorawan-stack/blob/master/api/gateway.proto#L48-L53